docs/tutorial.md

```diff
@@ -171,7 +171,7 @@ The issue is that the CNOT expects a qubit as second input argument where an int
 
 ### Merging single qubit gates
 
-All single-qubit gates appearing in a circuit can be merged by applying `merge_single_qubit_gates()` to the circuit.
+All single-qubit gates appearing in a circuit can be merged by applying `merge(merger=SingleQubitGatesMerger())` to the circuit.
```

Comment:
```suggestion
All single-qubit gates appearing in a circuit can be merged by applying `merge` (with a given instantiation from `Merger`) to the circuit.
```

Reply:
In the future we may add a two-qubit gate merger as well. So this should be a specific instantiation of the `SingleQubitGatesMerger`, not just any kind of `Merger`